### ISSUE TITLE
[Copy] Null when copying with string constant after editing the string constants in the library properties

### ISSUE THREAD

**Issue by JXNCTED** [DD2480-Group1/jabref#13]
### JabRef version

Our development version 

### Operating system

GNU / Linux

### Checked with the latest development build (copy version output from About dialog)

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

1. I ran the program with idea's run
2. I paste the example entry by the user's issue
3. I manually input the string constant
4 . I copy the entry by clicking on it and Ctrl-C
5, Pasting it into an editor and the null occur
6. Rerun with the idea run, by closing the window and clicking the run again
7. Copying and paste works fine now
8. Add another dummy string constant
9. Null appears again

**Comment by JXNCTED** [comment]
![image](https://github.com/DD2480-Group1/jabref/assets/33616271/a8c22cdd-daa9-413a-ba2a-6f6982640957)

The parsedSerilization is null when setting the content.

**Comment by JXNCTED** [comment]
After adding on string constant, the copy result in 
```
null@Article{Claudepierre,
  author  = {Claudepierre, S. G.},
  journal = grl,
}
```

**Comment by andersblomqvist** [comment]
Closed by #14
